### DISCUSSION TITLE
Can I make custom counter?

### DISCUSSION THREAD

**Discussion by Bravemobin** [igordanchenko/yet-another-react-lightbox#179] [Q&A] [answered]
I use library for messenger 
I have few images in my array but should show custom number for counter because have more images

**Comment by igordanchenko** [comment]
Why don't you create `slides` array of a length equal to the total number of slides?

**Reply by Bravemobin** [reply]
how should I do this ?

**Reply by igordanchenko** [reply]
Do you know the total number of slides from the very beginning?

**Reply by Bravemobin** [reply]
yeah i know

**Reply by igordanchenko** [reply, marked as answer]
So it sounds like you should be able to create `slides` array of a length equal to the total number of images, and fill the first N elements of the array with the first batch of images.

**Reply by igordanchenko** [reply]
Does this answer your question? If you require further assistance, please post your code snippet or sandbox showing how you configure the lightbox and update your `slides` array.

**Reply by Bravemobin** [reply]
yeah I make array of slides and it fix it Thanks;
